docs/sources/enterprise/usage-insights/dashboard-datasource-insights.md

```diff
@@ -0,0 +1,43 @@
++++
+title = "Dashboard and data source insights"
+description = "Understand how your dashboards and data sources are used"
+keywords = ["grafana", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/dashboard-datasource-insights.md"]
+weight = 200
++++
+
+# Dashboard and data source insights
+
+For every dashboard and data source, you can access usage information.
+
+## Dashboard insights
+
+> **Note:** Available in Grafana Enterprise v7.0+.
+
+To see dashboard usage information, from the top bar, click **Dashboard insights**.
+
+{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_button.png" max-width="400px" class="docs-image--no-shadow" >}}
+
+Dashboard insights displays the following information:
+
+- **Stats:** Shows the daily query count and error count for the last 30 days.
+- **Users & activity:** Shows the daily view count for the last 30 days; last activities on the dashboard and recent users (with a limit of 20).
+
+{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_stats.png" max-width="400px" class="docs-image--no-shadow" >}}{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_users.png" max-width="400px" class="docs-image--no-shadow" >}}
+
+## Data source insights
+
+> **Note:** Available in Grafana Enterprise v7.3+.
+
+Data source insights give you information about how a data source has been used in the last thirty days.
+
+- Queries per day
+- Errors per day
+- Average load duration per day (ms)
```

Comment:
What does "load duration" mean?

Reply:
the amount of time each query takes to load, averaged over that day.

Reply:
Does it make more or less sense to use, "Query load time per day (averaged in ms)"?